### PR TITLE
AC 538: Add Toast Message when going online.

### PR DESCRIPTION
**Pull request by Abdelaty**
Signed-off-by: Abdelaty Mohammedmagdi Abdelaty <Abdelaty.mohammedmagdi@gmail.com>

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-538

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ACBaseActivity.java [hunk 1]
```diff
@@ -180,6 +180,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
                         PatientDetailsFragment.snackbar.dismiss();
                     if (snackbar != null)
                         snackbar.dismiss();
+                        ToastUtil.showShortToast(getApplicationContext(), ToastUtil.ToastType.SUCCESS, R.string.connected_to_server_message);
```
Comment (VANKINEENITAWRUN) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ACBaseActivity.java]: @Abdelaty please format code with Android studio formatter.  Remove the tab space before the  `ToastUtil.showShortToast` line.

**Comment by TarekAlabd** [comment]
> Good work

In addition, You can add a small comment before it to describe what it should do

File: openmrs-client/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -338,7 +338,7 @@
     <!-- Syncbutton toasts -->
     <string name="disconn_server">Disconnecting from server</string>
     <string name="reconn_server">Reconnecting to server</string>
-
+    <string name="connected_to_server_message">connected to the server successfully</string>
```
Comment (vansha10) [inline openmrs-client/src/main/res/values/strings.xml]:
```suggestion
    <string name="connected_to_server_message">Connected to the server successfully</string>
```
Apologies for being a Grammar Nazi, but following the convention in other string values, the C must be capital.

Comment (Abdelaty) [reply openmrs-client/src/main/res/values/strings.xml]: I got the previous review from @f4ww4z to make it in the lower case. @vansha10  Thank you for commenting.

**Comment by Abdelaty** [comment]
@f4ww4z  Any suggestions?

**Comment by Abdelaty** [comment]
> Good work @Abdelaty . Will merge this for v2.7.1 .

Thank you @f4ww4z 
Can you check my comments on those JIRA Issues
https://issues.openmrs.org/browse/AC-566
https://issues.openmrs.org/browse/AC-565

File: openmrs-client/src/main/res/values/strings.xml (hunk 2 above)
Comment (Abdo-A) [inline openmrs-client/src/main/res/values/strings.xml]: How about a more user friendly message?
Users aren't necessarily familiar with what a server is.
A better message in my opinion could be: 'You are now connected!'

Comment (f4ww4z) [reply openmrs-client/src/main/res/values/strings.xml]: @Abdo-A I agree with this. It's shorter and more concise. @Abdelaty can you make the changes?

**Comment by Abdelaty** [comment]
Sure,  I will change it

**Comment by Abdelaty** [comment]
MOVED TO [PR](https://github.com/openmrs/openmrs-contrib-android-client/pull/562)

**Pull request closed without merging**